roles/openshift_logging/README.md

```diff
@@ -63,8 +63,8 @@ When both `openshift_logging_install_logging` and `openshift_logging_upgrade_log
 - `openshift_logging_es_client_key`: The location of the client key Fluentd uses for openshift_logging_es_host. Defaults to '/etc/fluent/keys/key'.
 
 - `openshift_logging_es_cluster_size`: The number of ES cluster members. Defaults to '1'.
-- `openshift_logging_es_cpu_limit`:  The amount of CPU limit for the ES cluster.  Unused if not set
-- `openshift_logging_es_memory_limit`: The amount of RAM that should be assigned to ES. Defaults to '8Gi'.
+- `openshift_logging_es_cpu_limit`:  The amount of CPU limit for the ES cluster. Defaults to no limitation, unused if not set and lower bound is 1 core.
```

Comment:
Consider having this read: "The upper limit of CPU usage allowed for a given Elasticsearch pod. Defaults to unlimited; unused if not set; lower bound is 1 core."